shared/actions/config/index.js

```diff
@@ -68,7 +69,7 @@ export function bootstrap (): AsyncAction {
       })
     } else {
       Promise.all(
-        [dispatch(getCurrentStatus()), dispatch(getExtendedStatus()), dispatch(getConfig())]).then(() => {
+        [dispatch(getCurrentStatus()), dispatch(getExtendedStatus()), dispatch(getConfig()), dispatch(favoriteList())]).then(() => {
```

Comment:
Because there's no widget on mobile, favoriteList() action/RPC was never being called.  I put it into the general bootstrapping in config().  Happy to move it somewhere else if we prefer -- this seemed appropriate to me.